contracts/0.4.24/Lido.sol

```diff
@@ -210,7 +210,7 @@ contract Lido is ILido, IsContract, Pausable, AragonApp {
     function setWithdrawalCredentials(bytes _withdrawalCredentials) external auth(MANAGE_WITHDRAWAL_KEY) {
```

Comment:
Since we're already using `bytes32` storage for withdrawal credentials, I'd just change the argument type to `bytes32`. We can upgrade the contract if the spec changes.

Reply:
Done